e2e/my.test.js

```diff
@@ -0,0 +1,60 @@
+ const { log } = require('detox');
+
+describe('Example', () => {
+  beforeAll(async () => {
+    await device.launchApp();
+  });
+
+  beforeEach(async () => {
+    await device.reloadReactNative();
+  });
+
+  it('should have a visible button', async () => {
+    // log.error(device.appLaunchArgs.get())
+    await expect(element(by.traits(['button'])).atIndex(1)).toBeVisible();
+  });
+
+  it("New wallet button works", async () => {
+    let newWalletButton = element(by.text("CREATE NEW WALLET (NEW SEED)"));
+    await newWalletButton.tap();
+    await element(by.text("I HAVE SAVED \n THE SEED")).tap();
+    // For some reason, the element that says 'ZEC', according to react, 
+    // has text of ' ---'
```

Comment:
Well, in fact this field in the header have no sense at all, if you are creating a wallet obviously have 0 ZEC. it will disappear in a near future, ouuuuu, I don't know, IMO.